apps/web/src/components/safe-apps/AppFrame/index.tsx

```diff
@@ -30,8 +30,10 @@ import { useSanctionedAddress } from '@/hooks/useSanctionedAddress'
 import BlockedAddress from '@/components/common/BlockedAddress'
 import { isSafePassApp } from '@/features/walletconnect/services/utils'
 import { BRAND_NAME } from '@/config/constants'
+import { AppRoutes } from '@/config/routes'
 
 const UNKNOWN_APP_NAME = 'Unknown Safe App'
+const NON_QUEUE_BAR_ROUTES = [AppRoutes.bridge]
```

Comment:
Why is this needed for Bridging but not for Swaps? Are they using a different iframe component?

Reply:
> Are they using a different iframe component?

Yes, we [use their `CowSwapWidget` and access the `iframe` element](https://github.com/safe-global/safe-wallet-monorepo/blob/fix/bridge-ui/apps/web/src/features/swap/index.tsx#L287-L313) instead of embedding it ourselves in an `AppFrame`.

Reply:
What about Staking? I see it also uses the AppFrame. Does it currently show the queue bar?

Reply:
Yes.

![grafik](https://github.com/user-attachments/assets/076bfab5-8039-45f7-b64a-7c8393a6fd18)

Reply:
Should we unify this behavior and hide the queue based on the `isNativeEmbed` prop?

Reply:
Agree, I don't think we deliberately decided to show the queue bar in any of these pages so hiding it if `isNativeEmbed` makes sense to me as well.

Reply:
Changed in 04c2dde433709c61291082037e0bb83492eb25d6.